docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -177,7 +177,14 @@ Method | Type | Description
 ------ | ----- | -----
 `filter(index)` | Selector | Creates a selector that filters a matching set by `index`.
 `filter(cssSelector)` | Selector | Creates a selector that filters a matching set by `cssSelector`.
-`filter(filterFn)` | Selector | Creates a selector that filters a matching set by `filterFn`; `filterFn` is a [client function](../obtaining-data-from-the-client.md#creating-client-functions) predicate that receives a node.
+`filter(filterFn, dependencies)` | Selector | Creates a selector that filters a matching set by the `filterFn` predicate. Use an optional `dependencies` parameter to pass functions, variables or objects used in the `filterFn` function internally.
```

Comment:
I'm not sure that it's clear `what is it dependencies and how to use them?` from this short description.
Maybe we should add more wide description for it like [here](https://github.com/DevExpress/testcafe/blob/master/docs/articles/documentation/test-api/obtaining-data-from-the-client.md#optionsdependencies)?
/cc @VasilyStrelyaev